### ISSUE TITLE
Instructions no longer work on macOS 11.1: "mkdir: /Developer: Read-only file system"

### ISSUE THREAD

**Issue by juj** [devernay/xcodelegacy#61]
jukkaj-0A4M:xcodelegacy jukkaj$ sudo ./XcodeLegacy.sh -osx1013 install
Password:
*** Info: found Xcode >= 4.3 in /Applications/Xcode.app
*** installed Xcode1013SDK.tar.gz
*** modified MacOSX Info.plist
*** Warning: /Developer/SDKs should be a symlink to /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs
Check that /Developer exists, and fix /Developer/SDKs with:
 $ sudo ln -sf '/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs' /Developer/SDKs
jukkaj-0A4M:xcodelegacy jukkaj$ ls /Developer
ls: /Developer: No such file or directory
jukkaj-0A4M:xcodelegacy jukkaj$ mkdir /Developer
mkdir: /Developer: Read-only file system
jukkaj-0A4M:xcodelegacy jukkaj$ sudo mkdir /Developer
mkdir: /Developer: Read-only file system
jukkaj-0A4M:xcodelegacy jukkaj$

**Comment by devernay** [comment]
Yes, this is no longer necessary with recent macOS, so you an ignore that warning, but I don't know at which version it stopped being necessary. ANyone has an idea?

**Comment by ipatch** [comment]
i was able to appease this warning/error with the below info.

https://apple.stackexchange.com/questions/395508/can-i-mount-the-root-system-filesystem-as-writable-in-big-sur

**Comment by startergo** [comment]
Aha. And how to extract the Xcode from the `Xcode.mpkg`? The old installer will not install on the newer macs.
